modules/Bio/EnsEMBL/Compara/PipeConfig/ProteinTrees_conf.pm

```diff
@@ -251,10 +253,41 @@ sub default_options {
     #default maximum retry count:
         'hive_default_max_retry_count' => 1,
 
+        # parameters for OrthologQMAlignment
+        'species1'         => undef,
+        'species2'         => undef,
+        'species_set_name' => "collection-" . $self->o('collection'),
+        'species_set_id'   => undef,
+        'ref_species'      => undef,
+        'homology_method_link_types' => ['ENSEMBL_ORTHOLOGUES'],
```

Comment:
This should be set to `['ENSEMBL_ORTHOLOGS', 'ENSEMBL_HOMOEOLOGS']` either here or in the plants configs